### ISSUE TITLE
Contract tests:

### ISSUE THREAD

**Issue by ibudasov** [new2lagoon/dev.lagoon#2]
- tests suppose the test the service contract: the statements which are in the service contract document, in our case it's swagger file. 
- create a separate namespace for this kind of tests
- the test setup should parse the swagger file, and call all the possible endpoints which are described in the file
- there suppose to be a library to do that, make an investigation
-------------------------------------------------------
Igor Budasov, [25.02.19 14:45]
so, the main idea - to create a test, which gonna get all the endpoints from the swagger file, and make requests to them, and receive the response code and compare it with swagger file again

Igor Budasov, [25.02.19 14:46]
basically, here you're going to test the swagger file and if the information in it is correct

Igor Budasov, [25.02.19 14:46]
it will be one test, you will not need to add manually the tests there
